pre_commit/util.py

```diff
@@ -255,7 +255,7 @@ def handle_remove_readonly(
         excvalue = exc[1]
         if (
                 func in (os.rmdir, os.remove, os.unlink) and
-                excvalue.errno == errno.EACCES
+                excvalue.errno in (errno.EACCES, errno.EPERM)
```

Comment:
just a minor suggestion: it is better to use sets for `in` comparisons instead of lists/tuples:

```suggestion
                excvalue.errno {errno.EACCES, errno.EPERM}
```